src/main/java/core/basesyntax/model/User.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.model;
+
+import java.util.List;
+
+public class User {
+
+    private String name;
+    private List<Bet> bets;
```

Comment:
```suggestion
    private int age;
```
let's not bind our entities for now